### DISCUSSION TITLE
Hono Context

### DISCUSSION THREAD

**Discussion by shouryadey** [honojs/hono#1027] [Q&A]
Hey, is there any documentation on Context Object used in Hono. The info provided here does not seem sufficient.
https://hono.dev/api/context

**Comment by yusukebe** [comment]
Hi @shouryadey !

You are right. I don't think there is enough documentation.

If there is something you would like to know about `Context`, we will describe it. It would also be helpful if you could make a PR for the addition to the [website](https://github.com/honojs/website) repository.
